RadarSDK/RadarVerificationManager.m

```diff
@@ -328,4 +337,233 @@ - (void)getAttestationWithNonce:(NSString *)nonce completionHandler:(RadarVerifi
+        @"/jb/jailbreakd.plist",
+        @"/jb/amfid_payload.dylib",
+        @"/jb/libjailbreak.dylib",
+        @"/usr/libexec/cydia/firmware.sh",
+        @"/var/lib/cydia",
+        @"/etc/apt",
+        @"/private/var/lib/apt",
+        @"/private/var/Users/",
+        @"/var/log/apt",
+        @"/Applications/Cydia.app",
+        @"/private/var/stash",
+        @"/private/var/lib/apt/",
+        @"/private/var/lib/cydia",
+        @"/private/var/cache/apt/",
+        @"/private/var/log/syslog",
+        @"/private/var/tmp/cydia.log",
+        @"/Applications/Icy.app",
+        @"/Applications/MxTube.app",
+        @"/Applications/RockApp.app",
+        @"/Applications/blackra1n.app",
+        @"/Applications/SBSettings.app",
+        @"/Applications/FakeCarrier.app",
+        @"/Applications/WinterBoard.app",
+        @"/Applications/IntelliScreen.app",
+        @"/private/var/mobile/Library/SBSettings/Themes",
+        @"/Library/MobileSubstrate/CydiaSubstrate.dylib",
+        @"/System/Library/LaunchDaemons/com.ikey.bbot.plist",
+        @"/Library/MobileSubstrate/DynamicLibraries/Veency.plist",
+        @"/Library/MobileSubstrate/DynamicLibraries/LiveClock.plist",
+        @"/System/Library/LaunchDaemons/com.saurik.Cydia.Startup.plist",
+        @"/Applications/Sileo.app",
+        @"/var/binpack",
+        @"/Library/PreferenceBundles/LibertyPref.bundle",
+        @"/Library/PreferenceBundles/ShadowPreferences.bundle",
+        @"/Library/PreferenceBundles/ABypassPrefs.bundle",
+        @"/Library/PreferenceBundles/FlyJBPrefs.bundle",
+        @"/Library/PreferenceBundles/Cephei.bundle",
+        @"/Library/PreferenceBundles/SubstitutePrefs.bundle",
+        @"/Library/PreferenceBundles/libhbangprefs.bundle",
+        @"/usr/lib/libhooker.dylib",
+        @"/usr/lib/libsubstitute.dylib",
+        @"/usr/lib/substrate",
+        @"/usr/lib/TweakInject",
+        @"/var/binpack/Applications/loader.app",
+        @"/Applications/FlyJB.app",
+        @"/Applications/Zebra.app",
+        @"/Library/BawAppie/ABypass",
+        @"/Library/MobileSubstrate/DynamicLibraries/SSLKillSwitch2.plist",
+        @"/Library/MobileSubstrate/DynamicLibraries/PreferenceLoader.plist",
+        @"/Library/MobileSubstrate/DynamicLibraries/PreferenceLoader.dylib",
+        @"/Library/MobileSubstrate/DynamicLibraries",
+        @"/var/mobile/Library/Preferences/me.jjolano.shadow.plist"
+    ]];
+    if (![RadarUtils isSimulator]) {
+        [suspiciousFiles addObjectsFromArray:@[
+            @"/bin/bash",
+            @"/usr/sbin/sshd",
+            @"/usr/libexec/ssh-keysign",
+            @"/bin/sh",
+            @"/etc/ssh/sshd_config",
+            @"/usr/libexec/sftp-server",
+            @"/usr/bin/ssh"
+        ]];
+    }
+    for (NSString *file in suspiciousFiles) {
+        if ([fileManager fileExistsAtPath:file]) {
+            [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Jailbreak check failed: File check"];
```

Comment:
worth adding the failed file path to the log?

Reply:
Will do